### PR TITLE
mesh: fix problem with infinite processing local network beacons 

### PR DESCRIPTION
**Pull request by przemyslawfierek-silvair**

### CODE REVIEW

File: mesh/net.c [hunk 1]
```diff
@@ -2619,13 +2621,16 @@ static void process_beacon(void *net_ptr, void *user_data)
 	 */
 	if (net->iv_upd_state == IV_UPD_INIT || ivi != net->iv_index ||
 							ivu != net->iv_update)
-		update_iv_ivu_state(net, ivi, ivu);
+		updated = update_iv_ivu_state(net, ivi, ivu);
 
 	if (kr != local_kr)
 		update_kr_state(subnet, kr, beacon_data->key_id);
```
Comment (michallowasrzechonek-silvair) [inline mesh/net.c]: Why not here as well?

Comment (przemyslawfierek-silvair) [reply mesh/net.c]: I had to read about `Key Refresh` mechanism. I've fixed it.

**Review by michallowasrzechonek-silvair — APPROVED** [review]
👏🏻 

Just clean up the history please ;)

**Comment by przemyslawfierek-silvair** [comment]
Duplicates: https://github.com/homersoft/bluez/pull/81 (created to connect branch with JIRA task)

**Pull request closed without merging**